### ISSUE TITLE
page_builder#add method sets an unexpected value in columns that are not in an argument array

### ISSUE THREAD

**Issue by muziyoshiz** [embulk/embulk#220]
In Embulk v0.6.14, page_builder#add method sets an unexpected value in the case below.

Example case:
7 columns were defined in self.transaction method.

```
        columns = [
          Column.new(0, "time", :timestamp),
          Column.new(1, "log_level", :string),
          Column.new(2, "message", :string),
          Column.new(3, "second_line", :string),
          Column.new(4, "third_line", :string),
          Column.new(5, "caused_by", :string),
          Column.new(6, "caused_by_message", :string),
        ]
```

3 values were inserted by page_builder#add method in run method.

```
        page_builder.add([ timestamp, log_level, message ])
```

And then unexpected values were set into 4 columns (i.e., second_line, third_line, caused_by and caused_by_message column) that were not in the argument of page_builder#add method. In previous version of Embulk, these columns were filled by nil.

For example, when I parsed the following example log by the following example parser, the 4 columns were filled by "ERROR".

Example log:
https://gist.github.com/muziyoshiz/f84b7a0cb367812b0055

Example parser:
https://github.com/muziyoshiz/embulk-parser-multiline-log-sample/blob/master/lib/embulk/parser/multiline-log-sample.rb

```
*************************** 1 ***************************
             time (timestamp) : 2015-03-14 11:12:22.123 UTC
        log_level (   string) : ERROR
          message (   string) : Book reader error
      second_line (   string) : Exception in thread "main" java.lang.IllegalStateException: A book has a null property
       third_line (   string) : at com.example.myproject.Author.getBookIds(Author.java:38)
        caused_by (   string) : java.lang.NullPointerException
caused_by_message (   string) :
*************************** 2 ***************************
             time (timestamp) : 2015-03-14 11:13:34.456 UTC
        log_level (   string) : WARN
          message (   string) : Suspected SQL injection attack (input="; DROP TABLE USERS --")
      second_line (   string) : ERROR
       third_line (   string) : ERROR
        caused_by (   string) : ERROR
caused_by_message (   string) : ERROR
*************************** 3 ***************************
             time (timestamp) : 2015-03-14 11:16:00 UTC
        log_level (   string) : ERROR
          message (   string) : Indescribable error
      second_line (   string) : ERROR
       third_line (   string) : ERROR
        caused_by (   string) : ERROR
caused_by_message (   string) : ERROR
*************************** 4 ***************************
             time (timestamp) : 2015-03-14 11:16:45.183 UTC
        log_level (   string) : ERROR
          message (   string) : Book reader error
      second_line (   string) : Exception in thread "main" java.lang.IllegalStateException: A book has a null property
       third_line (   string) : at com.example.myproject.Author.getBookIds(Author.java:38)
        caused_by (   string) : java.sql.SQLException
caused_by_message (   string) : Unknown reason
```

**Comment by frsyuki** [comment]
Thank you for your detailed report! I fixed this issue at #221.

**Comment by muziyoshiz** [comment]
Thank you! I confirmed the problem was solved by Embulk v0.6.15 in my environment.
